### DISCUSSION TITLE
Regulator representation between OpenDSS and conversion XML file

### DISCUSSION THREAD

**Discussion by motareddy** [GRIDAPPSD/GOSS-GridAPPS-D#1769] [Q&A] [answered]
Hi,

I am working on regulator control using VVO. At present working on an IEEE 123 feeder inside the GridAPPS-D platform. It seems like out of 4 regulators, three regulators can control when I noticed the model dictionary file. However, the representation of one of the regulators was quite confusing when I compared it with the OpenDSS file. Can anyone help me find the correct detail representation inside GridAPPS-D?

**Comment by AAndersn** [comment]
@shpoudel and @afisher1

**Comment by shpoudel** [comment]
What was the confusion when you compared it with OpenDSS? Please note that 3 out of 4 transformers have tank names and the substation regulator (reg1a) is defined as a PowerTransformer in CIM.

**Comment by motareddy** [comment]
One transformer of a regulator is designated in the model dictionary file as reg4 and its vgrp. From the OpenDSS file, I noticed reg4 used three single-phase transformers but in GridAPPS-D it reflected in the star connection in "vgrp" and reg3 and reg2 regulators "vgrp" is  Ii.

**Comment by shpoudel** [comment]
Could you clarify what the confusion is? reg4 is a three-phase regulator defined as three single-phase regulators. It means you can control the individual phases. Why are you particularly interested in vectorGroup?

**Comment by shpoudel** [comment, marked as answer]
This is not an issue with OpenDSS or CIM. The issue was within the optimization module for the VVO application.
